app/models/user.rb

```diff
@@ -47,6 +47,10 @@ def menu(scope = User.all)
     def by_name
       order(name: :asc)
     end
+
+    def roles_without_global_administrator
+      roles.except("global_administrator")
+    end
```

Comment:
There might possibly be multiple roles within the BopsConfig engine eventually so lets leave some space like this:

``` ruby
GLOBAL_ROLES = %w[global_administrator].freeze

def local_roles
  roles.keys - global_roles
end

def global_roles
  GLOBAL_ROLES
end
```

Reply:
Could also add a validation to ensure that someone doesn't hack a form:

``` ruby
validates :role, inclusion: {in: :local_roles}, if: -> { local_authority.present? } 
```